docs/installation.md

```diff
@@ -38,9 +38,9 @@ distribution is installed.
 
 ### Backend database
 
-Trino Gateway requires a MySQL or PostgreSQL database. Database initialization
-is performed automatically when the Trino Gateway process starts. Migrations
-are performed using `Flyway`.
+Trino Gateway requires a MySQL, PostgreSQL, or Oracle (12c or greater) database.
```

Comment:
Why is the minimum version 12c? TestDatabaseMigrationsOracle uses 18.4.0. We shouldn't mention untested versions here.

Reply:
Minimum version is 12c because the `FETCH` clause was not supported in Oracle in versions before that.

Reply:
https://endoflife.date/oracle-database mentions 12c already reached EOL. Ther is no need to document the version in my opinion.

Reply:
@willmostly @mosabua what do you think? Can we removed the version number for Oracle in the docs? we don't document MySQL or postgresql versions so it would be consistent to not mention a version.

Reply:
Yeah ... ultimately we probably should add them for all .. but that can wait until a follow up PR .. for now you can leave it out.

Reply:
+1